interfaces/README.md

```diff
@@ -2,5 +2,4 @@
 
 ## IBF dashboard
```

Comment:
is there a point to this readme? Given there's no point to the subfolder IBF-dashboard here (as we only have one interface), we could maybe get rid of this readme at least? or add an '# Interfaces' at least, similar to services/README.md

Reply:
I prefer to remove the readme and the `services` and `interfaces`. There is no point to keep a directory with only one sub-directory. However, it seemed too big a change to fit within this task.

There is a '# Interfaces'. Here it is https://github.com/rodekruis/IBF-system/blob/0c22c02533ebf4373b80e9a2919cc609abe02f18/interfaces/README.md#interfaces